### PR TITLE
Propose new files and consolidate the method

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

- new files in joomla 4 and joomla 3 
- add composer change
- add new check method

#### Testing Instructions

Tests should still passes and the github label should still work.

#### Additional info

I have not yet created the "composer Change" category in the tracker. If we agree on that proposal here we need to add it and maybe also fix the ids before this can be merged.

### CODE REVIEW

**Comment by zero-24** [comment]
The travis thing is expected as this is a result of the new `Composer Change` Category. Which is not final unitil there is a decision about the new method 😄

File: src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php [hunk 1]
```diff
@@ -231,6 +244,12 @@ class JoomlacmsPullsListener extends AbstractListener
 			'^administrator/components/com_fields',
 			'^components/com_fields',
 		],
+		// Composer Change
+		'72' => [
```
Comment (nibra) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: These 'magic numbers' (the array keys)  should be replaced with constants (or strings) to increase readability and maintainability.

Comment (zero-24) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: And than we add another string => id conversion to get the category id? Please send me a complete PR with your suggestions. Thanks.

Comment (nibra) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: Done.
Why it helps to improve? Well, numbers are easily miss-typed without any chance to notice (they still are correct numbers). With constants, that can not happen, and you get code completion for the possible values. You can understand their meaning immediately without having to consult a list with the description of numbers.

Comment (zero-24) [inline src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php]: Thanks just merged the PR.

**Comment by mbabker** [comment]
Change looks fine.  Let's get the test fixed and the category added on the site now.

**Comment by zero-24** [comment]
travis is happy now.

**Pull request merged**